components/jeune/TableauJeunes.tsx

```diff
@@ -285,11 +286,27 @@ export const TableauJeunes = ({
               <Link href={`/mes-jeunes/${jeune.id}`} key={jeune.id}>
                 <a
                   role='row'
-                  aria-label={`Accéder à la fiche de ${jeune.firstName} ${jeune.lastName}, dernière activité ${jeune.lastActivity}, ${jeune.messagesNonLus} messages non lus`}
+                  aria-label={`Accéder à la fiche de ${jeune.prenom} ${jeune.nom}, dernière activité ${jeune.lastActivity}, ${jeune.messagesNonLus} messages non lus`}
                   className={`table-row text-sm  items-center hover:bg-primary_lighten`}
                 >
                   <span role='cell' className='table-cell p-4'>
-                    {getJeuneFullname(jeune)}
+                    <span className='flex items-baseline'>
```

Comment:
```suggestion
                    <span className='flex items-center'>
```
pour aligner
![Capture d’écran 2022-06-20 à 15 50 34](https://user-images.githubusercontent.com/62615060/174616497-44962179-d109-45a9-a2f0-47d354663457.png)
r